concent_api/core/models.py

```diff
@@ -765,9 +765,9 @@ def clean(self) -> None:
                                           f'(requestor, provider or Concent) must be string and must be exactly '
                                           f'{ETHEREUM_ADDRESS_LENGTH} characters long.'
             })
-        if not isinstance(self.amount, int) or self.amount <= 0:
```

Comment:
Same here about the type, the message is ok here.
Btw. please just add dots at the end of the sentences, I see that sometimes they are and sometimes not, lets keep it consistent (and correct). Could be separate commit.

Reply:
Type not needed, dot added.